From: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>
Date: Thu, 30 May 2024 14:30:25 +0200
Subject: packaging: and needs-internet test restriction

Internet is required to pull Go dependencies to build and install spread, as
well as to download snap packages from the store.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>
---
 packaging/debian-sid/tests/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packaging/debian-sid/tests/control b/packaging/debian-sid/tests/control
index 724473d..4743e23 100644
--- a/packaging/debian-sid/tests/control
+++ b/packaging/debian-sid/tests/control
@@ -1,5 +1,5 @@
 Tests: integrationtests
-Restrictions: allow-stderr, rw-build-tree, needs-root, breaks-testbed, isolation-machine, needs-sudo
+Restrictions: allow-stderr, rw-build-tree, needs-root, breaks-testbed, isolation-machine, needs-sudo, needs-internet
 Depends: bzr,
          ca-certificates,
          git,
